### ISSUE TITLE
Missing bullet not appearing on homepage content [S]

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#927]
<img width="742" alt="Screen Shot 2020-06-04 at 5 30 31 PM" src="https://user-images.githubusercontent.com/38330843/83812213-1f0e9080-a689-11ea-9e7b-004fb2007c40.png">

https://docs.google.com/document/d/18W-nKAH-54wpNy5XSZeEA31my4XRMxpZRVJ3en4ThAA/edit
